core/Stream/hsStream.cpp

```diff
@@ -347,15 +353,58 @@ void hsStream::writeLine(const ST::string& ln, bool winEOL)
 /* hsFileStream */
 bool hsFileStream::FileExists(const ST::string& file)
 {
+#ifdef WIN32
+    HANDLE eFile = CreateFileW(file.to_wchar().data(), GENERIC_READ,
+        FILE_SHARE_READ | FILE_SHARE_WRITE, nullptr, OPEN_EXISTING, FILE_ATTRIBUTE_NORMAL, nullptr);
```

Comment:
I was thinking here that it wasn't really important to lock the file for writing as much as it would be in `hsFileStream::open()`. I wouldn't mind changing either this or `hsFileStream::open()` though.

Reply:
Makes sense 👍